File: ExampleApp/app/src/main/java/com/motiv/example/MainActivityContract.java
package com.motiv.example;

import androidx.fragment.app.*;
import dagger.*;
import dagger.android.*;
import dagger.android.support.*;
import javax.inject.*;

public interface MainActivityContract {

    public interface View {

        public java.lang.CharSequence edittext10getText();

        public void showToast(java.lang.String arg0);

        public void navigationControllerstartDrawerDashboard(com.motiv.example.User arg0);
    }

    public interface Presenter {

        public void eloonClick(android.view.View argument0);
    }
}

File: ExampleApp/app/src/main/java/com/motiv/example/MainActivityContract.kt
 
package com.motiv.example
import androidx.fragment.app.*
import dagger.*
import dagger.android.*
import dagger.android.support.*
import javax.inject.*

public interface MainActivityContract {

    public interface View {

        public fun edittext10getText(): CharSequence
        public fun showToast(arg0: String): Unit
        public fun navigationControllerstartDrawerDashboard(arg0: com.motiv.example.User): Unit
    }

    public interface Presenter {

        public fun eloonClick(argument0: android.view.View): Unit
    }
}
